### PR TITLE
Fix call to recaptcha_get_html() from ezcomFunctionCollection::fetchRecaptchaHTML()

### PR DESCRIPTION
**Pull request by mikaelcom**
The parenthesis was in the wrong place, isn't it ?

### CODE REVIEW

**Comment by crevillo** [comment]
+1

**Comment by mikaelcom** [comment]
Fix for EZP-21449

**Comment by nfrp** [comment]
**+1** for this, thanks for the contribution Mikaël!

File: packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php [hunk 1]
```diff
@@ -86,8 +86,8 @@ public static function fetchRecaptchaHTML()
         {
             $useSSL = true;
         }
-        return array( 'result' => recaptcha_get_html( $publicKey ), null, $useSSL );
+        return array( 'result' => recaptcha_get_html( $publicKey , null, $useSSL ));
```
Comment (andrerom) [inline packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php:89]: cs: should be space between the end paragraphs

Comment (nfrp) [inline packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php:89]: Did you mean the end-of-line parenthesis?

Comment (andrerom) [inline packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php:89]: yes :)

Comment (nfrp) [inline packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php:89]: Right :)

@mikaelcom : do you think you could push a new tiny pull-request to fix this little coding standards glitch?

Many thanks in advance!

**Comment by mikaelcom** [comment]
Do you mean  return array( 'result' => recaptcha_get_html( $publicKey , null, $useSSL ) ); ?
I'll do it as soon as I'm on my computer ;)

**Comment by nfrp** [comment]
@mikaelcom that is correct! Thank you very much!

File: packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php (hunk 1 above)
Comment (mikaelcom) [inline packages/ezcomments_extension/ezextension/ezcomments/classes/ezcomfunctioncollection.php:89]: see pull request #36

**Pull request merged**